CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# TBD
+
+## Enhancements
+
+- Update appium capabilities for appium 2.0 compatibility [512](https://github.com/bugsnag/maze-runner/pull/512)
```

Comment:
I still haven't released 7.27.0, so you could add this into there and bump the date.